ginivision/src/doc/source/event-tracking.rst

```diff
@@ -0,0 +1,60 @@
+Event Tracking
+====
+
+In version `3.12.0 <https://github.com/gini/gini-vision-lib-android/releases/tag/3.12.0>`_ we introduced the possibility to track various events which occur during the usage 
+of the Gini Vision Library.
+
+To subscribe to the events you need to implement the ``EventTracker`` interface and pass it to the
+builder when creating a new ``GiniVision`` instance.
+
+.. code-block:: java
+
+    GiniVision.newInstance()
+        .setEventTracker(new MyEventTracker());
+        .build();
+
+In ``MyEventTracker`` you can handle the events you are interested in.
+
+.. code-block:: java
+
+    class MyEventTracker implements EventTracker {
+
+        @Override
+        public void onCameraScreenEvent(final Event<CameraScreenEvent> event) {
+            switch (event.getType()) {
+                case TAKE_PICTURE:
+                    // handle the picture taken event
+                    break;
+                case HELP:
+                    // handle the show help event
+                    break;
+                case EXIT:
+                    // handle the exit event
+                    break;
+            }
+        }
+
+        @Override
+        public void onOnboardingScreenEvent(final Event<OnboardingScreenEvent> event) {
+            (...)
+        }
+
+        @Override
+        public void onAnalysisScreenEvent(final Event<AnalysisScreenEvent> event) {
+            (...)
+        }
+
+        @Override
+        public void onReviewScreenEvent(final Event<ReviewScreenEvent> event) {
+            (...)
+        }
+
+    }
+
+.. note::
+
+     If you use the Screen API all events will be triggered automatically.
+
+     If you use the Component API some events will not be triggered (for ex. events which rely on ``Activity#onBackPressed()``). You need to check whether all the events you are interested in are triggered.
```

Comment:
I might soften the "You need to check" bit and point the reader to where they can check.

Reply:
Done. Also on second thought I found the table of events you created really useful and added one as well. Had to tweak it to fit and also added a column for the API to quickly see in which API each is triggered automatically.